packages/image_picker/lib/image_picker.dart

```diff
@@ -6,31 +6,20 @@ import 'dart:async';
 import 'dart:io';
 
 import 'package:flutter/services.dart';
+import 'package:meta/meta.dart';
 
 /// Specifies the source where the picked image should come from.
 enum ImageSource {
-  /// Let the user choose an image from a source they prefer.
-  ///
-  /// On Android, opens a new screen with a grid of images (from the users
-  /// gallery) and a camera icon on the top right corner. The user can
-  /// either pick an image from the image grid, or tap the camera icon
-  /// to take a picture using the device camera.
-  ///
-  /// On iOS, the user is presented with an alert box with options to
-  /// either take a photo using the device camera or pick an image from
-  /// the photo library.
-  askUser,
```

Comment:
Removing this feature (and thus the default value) is a breaking change - I think it would be a good idea to send an email to flutter-dev@googlegroups.com letting people know.

Reply:
Originally there was no way of choosing a source for the images. The native sides of this plugin had the "source chooser dialogs" on iOS and Android. I introduced an option of controlling the image source in #271, and I think I should've removed the dialogs back then, but I didn't want to break anything.

(that was just me rationalizing why I did this breaking change :))

I can do the email when/if this PR gets approved, thanks for the idea!

Reply:
We prefer to warn people in advance of landing the PRs -- see https://flutter.io/design-principles/#handling-breaking-changes

Reply:
Right. I just read it through and that makes a lot of sense, sorry for my ignorance.

I _think_ I just sent the email, but can't see it anywhere. Can you verify it actually went through?